modules/custom/az_metrics/az_metrics.module

```diff
@@ -21,9 +21,14 @@ function az_metrics_data() {
   foreach ($domainQuery as $d) {
     array_push($domainList, $d->domain);
   }
-
+  // Fetch information about installation profile.
+  $information = $extension_list->getExtensionInfo('az_quickstart');
+  if (isset($information['version'])) {
+    $version = $information['version'];
+  }
   $data = [
     'domains' => $domainList,
+    'version' => $version,
```

Comment:
What are our coding standards for declaring an empty variable? 
$version is not currently in scope to be available since it's being defined inside of the if statement above.